apps/sms/services/js/message_manager.js

```diff
@@ -137,28 +187,39 @@ var MessageManager = {
     };
   },
 
+  /**
+   * Function to get the message object having a given id.
```

Comment:
You can check MDN docs (mdn.io/mozMobileMessage) too, where methods have nice explanation. you can take input from there also :)